recipes/recipes_emscripten/freetype/build.sh

```diff
@@ -1,5 +1,9 @@
 mkdir build && cd build
 
-emcmake cmake .. $CMAKE_ARGS
+# NOTE: freetype needs to be compiled with atomics/bulk-memory features
+# for use cases like cairo, pango etc.
```

Comment:
As can be seen here (https://github.com/emscripten-forge/recipes/actions/runs/11515675545/job/32056803985?pr=1396#step:9:1698)